src/olympia/files/models.py

```diff
@@ -607,6 +611,14 @@ def pretty_name(self):
             return parts[1]
         return self.name
 
+    @property
+    def channel(self):
+        return (
+            amo.RELEASE_CHANNEL_UNLISTED
+            if self.automated_signing
```

Comment:
Ugh, we should file an issue to rename + migrate that to `channel`...

Reply:
filed https://github.com/mozilla/addons-server/issues/17911